### PR TITLE
Include Banner

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**
Following instructions from last Solid Team meeting with minutes on https://www.w3.org/community/solid/wiki/Meetings#2020-07-15

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,6 +64,21 @@ <h1>Solid</h1>
...
 <!-- /headerwrap -->
 
+	    <div id="draft-warning"></div>
+<script>
+  if (document.location.hostname === 'localhost' || document.location.hostname === 'solid.github.io') {
+    const draftWarningElement = document.getElementById('draft-warning')
+    draftWarningElement.innerHTML = `
+      <div class="message is-danger is-large" role="alert">
+        <div class="message-body">
+          You are currently viewing the historical site of the Solid project which is was incepted at MIT. If you are looking for reliable information, visit the live site at <a href="https://solidproject.org/" title="The official Solid website">SolidProject.org</a>.
```
Comment (justinwb) [inline index.html]: Suggest a slight copy change:

You are viewing the historical site of the Solid MIT Project. Visit the <a href="https://solidproject.org">official Solid website</a> for reliable and up-to-date information.

**Review by justinwb — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (csarven) [reply index.html]: Perhaps:

You are viewing the original website of the Solid Project at MIT. Please join us at the official Solid project website for current activities.

Comment (justinwb) [reply index.html]: FWIW - I'm good with either suggestion (assuming we add a link on "official Solid project website" in sarven's version)

**Comment by justinwb** [comment]
> FWIW - I'm good with either suggestion (assuming we add a link on "official Solid project website" in sarven's version)

Merged version didn't actually link to solidproject.org

**Comment by Mitzi-Laszlo** [comment]
https://github.com/solid/solid.mit.edu/pull/31

**Comment by Mitzi-Laszlo** [comment]
how do we get it to actually show on solid.mit.edu?

**Pull request merged**